Sources/UITextView+Placeholder.m

```diff
@@ -199,11 +199,8 @@ - (void)updatePlaceholderLabel {
         textContainerInset = UIEdgeInsetsMake(8, 0, 8, 0);
     }
 
-    CGFloat x = lineFragmentPadding + textContainerInset.left;
-    CGFloat y = textContainerInset.top;
-    CGFloat width = CGRectGetWidth(self.bounds) - x - lineFragmentPadding - textContainerInset.right;
-    CGFloat height = [self.placeholderLabel sizeThatFits:CGSizeMake(width, 0)].height;
-    self.placeholderLabel.frame = CGRectMake(x, y, width, height);
+    self.placeholderTextView.textContainer.exclusionPaths = self.textContainer.exclusionPaths;
+    self.placeholderTextView.frame = self.bounds;
```

Comment:
I think we should set `textContainerInset` to `placeholderTextView` as well. Tested on the demo project with:

```objc
textView.textContainerInset = UIEdgeInsetsMake(50, 0, 0, 100);
```

<img width="1118" alt="screenshot" src="https://user-images.githubusercontent.com/931655/40537552-31a194e0-604a-11e8-92b0-bf079789d0d4.png">